### PR TITLE
Adding to what's new for users

### PR DESCRIPTION
**Pull request by hflynn**
This page looked a bit sparse, especially given we are linking to it from the release announcement

To be staged at https://www.openmicroscopy.org/site/support/omero5.2-staging/users/whatsnew.html

### CODE REVIEW

**Comment by hflynn** [comment]
Building now

**Comment by jburel** [comment]
right-hand?

**Comment by jburel** [comment]
```Allow export of big images``` (limit is now 12kx12k)

File: omero/users/whatsnew.txt [hunk 1]
```diff
@@ -3,7 +3,13 @@ What's new for OMERO 5.2 for users
...
+-  improved support for screen data formats (amongst others) and import times
+   for images with many ROIs
+-  reworked display of metadata and annotations in the righthand pane of both
+   clients
 -  improved support for attaching analytical results
 -  significant improvement in client performance including for loading data
    and drag and drop actions
+
+Note that you can no longer launch the desktop clients via Java Web Start as
+we have dropped support for this due to technical issues beyond our control.
```
Comment (joshmoore) [inline omero/users/whatsnew.txt]: Link to blog post?

**Comment by hflynn** [comment]
Should be good now, I'll rebuild staging once Travis is green.

**Comment by sbesson** [comment]
Reads well to me. Ready to merge once @joshmoore and @jburel have signed off on the last commit.

**Comment by joshmoore** [comment]
WFM.

**Comment by jburel** [comment]
good to merge

**Pull request merged**